src/Controls/src/Core/Application/Application.cs

```diff
@@ -256,6 +256,7 @@ void TriggerThemeChangedActual()
 
 				OnParentResourcesChanged([new KeyValuePair<string, object>(AppThemeBinding.AppThemeResource, newTheme)]);
 				_weakEventManager.HandleEvent(this, new AppThemeChangedEventArgs(newTheme), nameof(RequestedThemeChanged));
+				OnPropertyChanged(nameof(UserAppTheme));
```

Comment:
One day I'll ask why application properties are not bindable properties... But not today!